Backend/Core/XMLParser.py

```diff
@@ -72,16 +72,118 @@ def convertISSDBKeyToXML(requestData):
     elem.append(dataChild)
     return elem
 
+
+# XML-Structure for AstrosOnISS
+# <Request>
+#   <requestName>AstrosOnISS</requestName>
+#   <data>
+#       <Astro name="max muster">
+#          <picture>link</picture>
+#          <flag>link</flag>
+#          <nation>link</nation>
+#       </Astro>
+#       <Astro name="max muster">
+#          <picture>link</picture>
+#          <flag>link</flag>
+#          <nation>link</nation>
+#       </Astro>
+#   </data>
+# </Request>'
+
+def convertAstrosToXML(requestData):
```

Comment:
Private function have to be declared with a _ at the beginning according to Python naming conventions. Please change that.

Reply:
done